final_task/rss_reader/functions/check_func.py

```diff
@@ -0,0 +1,52 @@
+"""Contain checking functions"""
+
+import requests
+import logging
+import sys
+import classes.exceptions as exc
+
+def check_internet_connection(logger):
+    """Check if network connection is avaliable"""
+    try:
+        logger.info("Checking internet connection...")
+        requests.get('https://www.google.com/', timeout=1)
+        return True
+    except requests.ConnectionError:
+        raise exc.InternetConnectionError("No internet access")
+
+
+def check_verbose(command_line_args):
+    """If vebose argument is True, set logging to info level"""
+
+    logging.basicConfig(format='[%(asctime)s][%(levelname)s] %(message)s',
+                        stream=sys.stdout,
+                        level=logging.INFO
+                        )
+    logger = logging.getLogger("rss_reader_logger")
+    if command_line_args.verbose:
+        logger.setLevel(level=logging.INFO)
+        return logger
+    else:
+        logger.setLevel(level=logging.WARNING)
+        return logger
+
+
+def check_feed_status(feed):
+    """Check HTTP status code"""
+
+    try:
+        status = feed.status
+        if 600 > status >= 400:
```

Comment:
There is a better way to check if it's 500 or 400 HTTP response. Please, rework this solution

Reply:
Fixed